grabbit/src/main/groovy/com/twcable/grabbit/GrabbitConfiguration.groovy

```diff
@@ -78,6 +79,23 @@ class GrabbitConfiguration {
         )
     }
 
+    private
+    static Collection<String> nonEmptyCollection(Collection<String> excludePaths, ConfigurationException.Builder errorBuilder) {
+        String prePattern = "^(/|\\./|\\|\\\\).*\$";
+        String postPattern = ".*(/|\\./|\\|\\\\)\$";
+        if (excludePaths == null || excludePaths.isEmpty())
+            return excludePaths
+        if (excludePaths.any({ String it -> (it == null) || (it.isEmpty()) })) {
+            errorBuilder.add('excludePaths', 'contains null/empty')
+            return excludePaths
+        } else if (excludePaths.any { String it -> it.matches(prePattern) } || excludePaths.any { String it -> it.matches(postPattern) }) {
+            errorBuilder.add('excludePaths', 'relative paths provided cannot begin or end with /,./ or \\')
```

Comment:
Can't `./` just be a no-op like on the file system?

e.g.   `path: "/content/a", excludePaths: ["./b"]` -> `"/content/a/./b"` -> `"/content/a/b"`